code/modules/clothing/shoes/wheelys.dm

```diff
@@ -26,7 +26,7 @@
 	if(!isliving(user))
 		return
 	if(!istype(user.get_item_by_slot(ITEM_SLOT_FEET), /obj/item/clothing/shoes/wheelys))
-		to_chat(user, span_warning("You must be wearing the wheely-heels to use them!"))
+		balloon_alert(user, "You must be wearing [src] to use [p_them()]!")
```

Comment:
Since you're opting to go with a balloon alert, they are formatted slightly different:
- Not capitalized
- No punctuation besides `!` to denote failures
- Concise and clear

As an example:
```suggestion
		balloon_alert(user, "must be worn!")
```

Reply:
Sounds good, thanks!